features/in_onderzoek.feature

```diff
@@ -0,0 +1,44 @@
+# language: nl
+
+Functionaliteit: in onderzoek
+  Wanneer een attribuut in onderzoek is, krijgt in het antwoord het attribuut met dezelfde naam binnen inOnderzoek de waarde true.
+
+  Een categorie kan in zijn geheel in onderzoek zijn, maar er kunnen ook individuele attributen binnen de categorie in onderzoek zijn.
+  Wanneer een categorie in zijn geheel in onderzoek is, krijgt in het antwoord in inOnderzoek elk attribuut van deze categorie de waarde true.
+  Wanneer een categorie in zijn geheel in onderzoek is, krijgt in het antwoord in  inOnderzoek in elke groep van de categorie elk attribuut van deze categorie de waarde true.
+
+
+  Een groep kan in zijn geheel in onderzoek zijn, maar er kunnen ook individuele attributen binnen de groep in onderzoek zijn.
+  Wanneer een groep in zijn geheel in onderzoek is, krijgt in het antwoord in inOnderzoek elk attribuut van deze groep de waarde true.
+
+  Een attribuut dat niet in onderzoek is, wordt niet in het antwoord in inOnderzoek opgenomen, ook niet met de waarde false of null.
+
+  Scenario: hele categorie reisdocument in onderzoek
+    Gegeven het te raadplegen reisdocument heeft het hele reisdocument in onderzoek (35.83.10=010000)
```

Comment:
Hoe kom je aan 35.83.10 en 010000?

Reisdocumenten is categorie 12, dus het onderzoek zit in element 12.83.10. Als de hele categorie reisdocument in onderzoek is, dan heeft 12.83.10 de waarde "120000"

Reply:
Aangepast